### DISCUSSION TITLE
The Manifold media player is inaccessible with standard keyboard commands

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3483] [Accessibility]
STEPS TO REPRODUCE:
1. Click on a resource annotation in Manifold, like this one: https://uw.manifoldapp.org/read/0fb62fe5-4ec4-438d-9692-ed5f078192f7/section/52047f4b-ef7d-4299-9b6a-15f5fe2021df/resource/a5258d64-b0bc-44d0-a8ab-96348e41708f
2. Using standard keyboard commands, like the tab key, try to play the video. 

OBSERVED BEHAVIOR:
I notice that as I hit the tab key, I cannot navigate to the play button in the media player. Instead, I skip over the media player and go right to the Download and Visit Resource Page buttons.

EXPECTED BEHAVIOR:
Using keyboard commands, I should be able to navigate to the controls in the Manifold media player--similar to what I can do with a YouTube video.

**Comment by dananjohnson** [comment]
We fixed this in v9. Internal video resources now use the native `<video>` element for playback.
